keyboards/inland/v83p/config.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 jonylee@hfd (@jonylee1986)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#define ENCODER_DEFAULT_POS 0x3
```

Comment:
Is this actually needed?

Reply:
yes,  without it the encoder will not respond during fast rotation @drashna